src/components/TutorialCarouselDialog/components/tutorial6.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Typography, withStyles } from 'material-ui';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+import styles from '../styles';
+
+@injectIntl
+@withStyles(styles, { withTheme: true })
+export default class Tutorial6 extends Component {
+  static propTypes = {
+    classes: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { classes } = this.props;
+
+    return (
+      <div>
+        <Typography variant="display1">
+          <FormattedMessage id="tutorial6.title" defaultMessage="Decentralization" />
+        </Typography>
+        <ul className={classes.contentList}>
+          <li>
+            <Typography variant="body2">
+              <FormattedMessage id="tutorial6.content0" defaultMessage="Having to be in control of your own security is the trade-off for decentralization." />
+            </Typography>
+          </li>
+          <li>
+            <Typography variant="body2">
+              <FormattedMessage id="tutorial6.content1" defaultMessage="You don't have to rely on your bank, government, or anyone else when you want to move your funds." />
+            </Typography>
+          </li>
+          <li>
+            <Typography variant="body2">
+              <FormattedMessage id="tutorial6.content2" defaultMessage="You don't have to rely on the security of an exchange or bank to keep your funds safe." />
+            </Typography>
+          </li>
+          <li>
```

Comment:
Can we please create a component that we use in all of these? Maybe something like:
```js
const Item = injectIntl(({ id, message }) => (
          <li>
            <Typography variant="body2">
              {intl.formatMessage({ id, message })}
            </Typography>
          </li>
))
```